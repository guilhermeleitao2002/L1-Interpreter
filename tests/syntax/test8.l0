let x = match y {
  Some(x) ->
};;