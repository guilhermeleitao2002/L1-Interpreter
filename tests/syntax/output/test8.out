Syntax Error:
ParseException: Encountered " "}" "} "" at line 3, column 1.
Was expecting one of:
    ...